test/integration/visitor_cannot_view_orders_and_admin_views_test.rb

```diff
@@ -87,7 +87,7 @@ class VisitorCannotViewOrdersAndAdminViewsTest < ActionDispatch::IntegrationTest
     create_and_login_additional_users(1)
     user = User.first
 
-    visit "/admin/dashboard/#{user.id}"
+    visit '/admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.